spec/requests/login_spec.rb

```diff
@@ -0,0 +1,48 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# NOTE: in deployed environments, this interaction is handled by Apache and
+#       Shibboleth in concert with the application
+RSpec.describe 'Login/out requests' do
```

Comment:
Can we delete spec/controllers/ now?

Reply:
No, I still need to try to convert that guy to a feature spec to test the "user return to" logic. (I'll do that next.)